angular-ngrx-scss/src/app/state/repository/repository.selectors.ts

```diff
@@ -1,6 +1,6 @@
 import { createFeatureSelector } from '@ngrx/store';
 import { RepoState } from './repository.state';
 
-export const repositoryFeatureKey = 'repository';
+export const repositoryFeatureKey = 'repo';
```

Comment:
suggestion: What's the reason for changing the feature key? I don't think this needs to be changed.

- [ ] Why is this adjusted?

Reply:
This was changed because the state slice is called "repo", so the initial version was not working till I updated the key.